pkg/p2p/wire/message/agreement.go

```diff
@@ -390,9 +394,15 @@ func UnmarshalStepVotes(r *bytes.Buffer) (*StepVotes, error) {
 // MarshalVotes marshals an array of StepVotes.
 func MarshalVotes(r *bytes.Buffer, votes []*StepVotes) error {
 	if err := encoding.WriteVarInt(r, uint64(len(votes))); err != nil {
+		log.WithError(err).Errorln("failed to marshal votes length")
 		return err
 	}
 
+	if len(votes) != 2 {
+		log.Infoln("failed to marshal step votes, 2 votes are required")
```

Comment:
Maybe we don't need this log (info) entry here as we already report an error at MarshalAgreement?